.github/workflows/source-parsers-lint.yaml

```diff
@@ -0,0 +1,20 @@
+name: P3 CI [xunit-autolabeler-v2/ast_parser]
+
+on:
+  push:
+    branches: [ master ]
+  pull_request:
+    branches: [ master ]
+
+jobs:
+  my-job:
+    name: Lint
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@v2
+    - name: CI build
+      env:
+        TEST_ENV: test
+        TRAMPOLINE_BUILD_FILE: ci-cd/p3_temp/lint_ast_parser.sh
```

Comment:
`p3` refers to `polyglot parser python`.

These are `temp` files because I'd like to run tests/lint against the `xunit-autolabeler-v2` directory _once all the pieces are merged + in place_. (Right now, these cover `xunit-autolabeler-v2/ast-parser` instead.)

Reply:
nit: explain acronyms before the first use